samples/AnomalyDetector/multivariate/models.tsp

```diff
@@ -342,7 +342,7 @@ model ModelList {
   maxCount: int32;
 
   @doc("The link to fetch more models.")
-  @nextLink
+  @Azure.Core.nextLink
```

Comment:
We already `using Azure.Core;` at the top, why do we still need qualified name here?

Reply:
> We already `using Azure.Core;` at the top, why do we still need qualified name here?

@chunyu3 Because it conflicts with `@TypeSpec.nextLink`...